src/main/integrations/gpu-context.ts

```diff
@@ -0,0 +1,76 @@
+import { defineIntegration, dropUndefinedKeys, Event } from '@sentry/core';
+import { app } from 'electron';
+
+interface Options {
+  infoLevel: 'basic' | 'complete';
+}
+
+interface GpuContext extends Record<string, unknown> {
+  name: string;
+  active?: boolean;
+  vendor_id?: string;
+  vendor_name?: string;
+  device_id?: string;
+  driver_version?: string;
+}
+
+interface GpuDevice {
+  active: boolean;
+  deviceId: number;
+  deviceString?: string;
+  driverVendor?: string;
+  driverVersion?: string;
+  vendorId: number;
+  vendorString?: string;
+}
+
+interface GpuInfoResult {
+  gpuDevice: GpuDevice[];
+}
+
+function gpuDeviceToGpuContext(device: GpuDevice): GpuContext {
+  return dropUndefinedKeys({
+    name: device.deviceString || 'GPU',
+    active: device.active,
+    vendor_id: `0x${device.vendorId.toString(16).padStart(4, '0')}`,
```

Comment:
Why do this transform?

Reply:
Device and vendor IDs are often referred to in hex notation.

For example the default windows display driver vendor ID is 0x1414. 

https://devicehunt.com/view/type/pci/vendor/1414

Saves users from having to the conversion before searching!